internal/gatewayserver/database/dbRepo/dbRepo.go

```diff
@@ -0,0 +1,32 @@
+package dbRepo
+
+import (
+	"context"
+
+	"github.com/razorpay/trino-gateway/pkg/spine"
+	"github.com/razorpay/trino-gateway/pkg/spine/db"
+)
+
+type DbRepo struct {
```

Comment:
This seems like a wrapper for `spine.Repo`. Why do we need this? Why can't we use `spine.Repo` directly? As it implements `IDbRepo` interface already?